sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -1694,6 +1747,40 @@ public partial class CassandraTableResourceInfo
         public Azure.ResourceManager.CosmosDB.Models.CassandraSchema Schema { get { throw null; } set { } }
         public string TableName { get { throw null; } set { } }
     }
+    public partial class ClientEncryptionIncludedPath
+    {
+        public ClientEncryptionIncludedPath(string path, string clientEncryptionKeyId, string encryptionType, string encryptionAlgorithm) { }
+        public string ClientEncryptionKeyId { get { throw null; } set { } }
+        public string EncryptionAlgorithm { get { throw null; } set { } }
+        public string EncryptionType { get { throw null; } set { } }
+        public string Path { get { throw null; } set { } }
+    }
+    public partial class ClientEncryptionKeyGetPropertiesResource : Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource
+    {
+        public ClientEncryptionKeyGetPropertiesResource() { }
+        public Azure.ETag? ETag { get { throw null; } }
```

Comment:
is the corresponding swagger correct that we have the `Etag` property in this `properties` level instead of top level of the resource model?

Reply:
@kr-santosh please take a look at this.

Reply:
yes this resource is associated with an etag

Reply:
All database resources have. Similar to public partial class SqlContainerGetPropertiesResource